SETUP.md

```diff
@@ -23,17 +23,16 @@ This document describes how to setup all the dependencies to run the notebooks i
 
 Depending on the type of recommender system and the notebook that needs to be run, there are different computational requirements. Currently, this repository supports the following environments:
 
-* Python CPU
```

Comment:
I have a similar concern to @yueguoguo, having `PySpark CPU / GPU` seems to indicate that we are doing pyspark on GPU. What about something like this:
* Python CPU
* Python GPU
* PySpark
* PySpark & Python GPU

Reply:
Would be better to add a "-" between Python and CPU/GPU. What do you @Microsoft/miguelgfierro @loomlike think?

Reply:
Make sense. I think Miguel's suggestion (4 bullets) better explains the actual environments. I would add env name as well. Please check new changes.

Reply:
In that sense, removing the bullet points look more simple to me.
> Currently, this repository supports **Python CPU**, **Python GPU** and **PySpark**.